src/rbd.cc

```diff
@@ -3158,6 +3158,34 @@ int main(int argc, const char **argv)
 	cerr << "rbd: order must be between 12 (4 KB) and 25 (32 MB)" << std::endl;
 	return EXIT_FAILURE;
       }
+      cerr << "rbd: --order is deprecated, please use --object-size" << std::endl;
+    } else if (ceph_argparse_witharg(args, i, &val, err, "--object-size", (char*)NULL)) {
+      if (!err.str().empty()) {
+	cerr << "rbd: " << err.str() << std::endl;
+	return EXIT_FAILURE;
+      }
+      int object_size = strict_sistrtoll(val.c_str(), &parse_err);
+      if (!parse_err.empty()) {
+        cerr << "rbd: error parsing --object-size " << parse_err << std::endl;
+        return EXIT_FAILURE;
+      }
+      // rbd: order must be between 12 (4 KB) and 25 (32 MB)
+      for (int i = 1; i < 27; i++) {
```

Comment:
If the goal is to round to the nearest order, you could also just perform ```std::round(std::log2(object_size))```.  Since this isn't performance critical code, the readability improvement is probably worth it.